### ISSUE TITLE
Search for ignored sync settings or extensions

### ISSUE THREAD

**Issue by sana-ajani** [microsoft/vscode#94034]
Testing https://github.com/microsoft/vscode/issues/93801

Changing extensions and settings that have been ignored to being synced again was smooth, but could we making searching for which extensions and settings easier?  Like being able to search for @sync: ignored in extensions view or in settings search

**Comment by roblourens** [comment]
There is already a setting, `sync.ignoredSettings` and you can see them all in one place there. But I like managing this through the context menu option too and I think this makes sense.

**Comment by roblourens** [comment]
What about the inverse to see synced extensions, would that be useful?

**Comment by sana-ajani** [comment]
Yeah, I think being able to search for all the extensions or settings that have been synced through the UI would be nice

**Comment by heartacker** [comment]
any updating? we need this  indeed
